staging/src/k8s.io/code-generator/cmd/client-gen/generators/client_generator.go

```diff
@@ -27,6 +27,7 @@ import (
 	"k8s.io/code-generator/cmd/client-gen/generators/util"
 	"k8s.io/code-generator/cmd/client-gen/path"
 	clientgentypes "k8s.io/code-generator/cmd/client-gen/types"
+	kubenamer "k8s.io/code-generator/pkg/namer"
```

Comment:
why the kube prefix?

Reply:
oh, ic. another import below

Reply:
Otherwise it collides with the `"k8s.io/gengo/namer"` package.
`kubenamer` is not a good alias, any recommendation?